docs/source/models/causes/other_causes/index.rst

```diff
@@ -74,16 +74,21 @@ States Data
      - 
    * - C
      - disabilty weight
-     - all_cause_yld_rate - anemia_yld_rate - population_maternal_disorders_yld_rate + pregnancy_maternal_disorders_yld_rate
+     - all_cause_yld_rate - anemia_yld_rate - population_maternal_disorders_yld_rate 
```

Comment:
Maybe I should have started here: what is the common-sense interpretation of continuing to subtract the population maternal disorders YLD rate, but not add in the pregnancy maternal disorders YLD rate?

Reply:
From our test runs, I realized that the majority of the impact that the inclusion of background morbidity had on YLDs averted in our sim was through the pathway of anemia YLDs averted causing increases in "other" YLDs by reducing comorbidities between them. This pathway was greater in magnitude than the effect of "other" YLDs reducing the number of anemia YLDs at baseline (only about a 1% reduction) AND greater than the pathway of accumulating YLDs due to other causes after an averted death (we only model 6 weeks after averted deaths). While anemia comorbidity only adjusts the other causes DW by ~1%, the magnitude of YLDs due to other causes is large, so it ends up being sizable relative to total YLDs averted between scenarios (~8%). 

Given that the maternal disorders YLD rate among the pregnant population is a significant portion of YLDs due to other causes as we had it written in the previous version, it would be overly conservative to apply increases associated with comorbid anemia YLD reductions to the full baseline amount of maternal disorders YLDs included in the other causes YLDs because this value should be reduced between scenarios in a way that we would not be modeling (because we only reduce the actual maternal disorders YLDs, not the ones implicitly included in background morbidity). So essentially, there would be a modeled 1% increase due to the reduction in comorbid anemia, but some unmodeled decrease as well. 

Removing maternal disorders YLDs from background morbidity removes this limitation from the largest pathway of influence on YLDs averted due to the inclusion of background morbidity in our model, although remains limited in other ways.

Reply:
Thanks for the explanation! I understand what this equation is saying now. In a way, we've gone back to the "default" approach to background morbidity that we'd be likely to use in other models -- we simply subtract out all the modeled causes from the all cause YLD rate. We're now ignoring the fact that we model maternal disorders in a special, unusual way, which I agree makes sense given your investigations into what is influential.

> This pathway was greater in magnitude than the effect of "other" YLDs reducing the number of anemia YLDs at baseline (only about a 1% reduction)

Does this make intuitive sense to you? I'm confused how this could be, since I assume that background morbidity YLDs are much larger than anemia YLDs, and it seems like we'd be almost multiplying anemia YLDs by (1 - background DW).

Reply:
Oddly, I've realized that the COMO adjustment does not depends on the magnitude of DW of both comorbid causes in ways that don't feel particularly intuitive. The COMO adjustment affects the magnitude of all comorbid causes equally and the influence is small because anemia DWs are small and it is fairly insensitive to the magnitude of the other causes DW.

I don't mean to open a can of worms in this comment thread, so let's plan to talk through all of this (maybe in a doc session?) when I'm back from being OOO!